vector/src/androidTest/java/im/vector/app/features/widgets/WebviewPermissionUtilsTest.kt

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (c) 2022 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package im.vector.app.features.widgets
+
+import android.Manifest
+import android.webkit.PermissionRequest
+import im.vector.app.InstrumentedTest
+import im.vector.app.features.widgets.webview.WebviewPermissionUtils
+import org.amshove.kluent.shouldBeEqualTo
+import org.junit.FixMethodOrder
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.JUnit4
+import org.junit.runners.MethodSorters
+
+@RunWith(JUnit4::class)
+@FixMethodOrder(MethodSorters.JVM)
+class WebviewPermissionUtilsTest : InstrumentedTest {
```

Comment:
I am not sure it should be intrumented test here. It could be only unit tests.

Reply:
Good call, made the change